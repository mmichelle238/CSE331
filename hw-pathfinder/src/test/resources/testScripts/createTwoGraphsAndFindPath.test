# create multiple graphs and find paths between them
# simultaneously.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

CreateGraph graph

AddNode graph A
AddNode graph B
AddNode graph C
AddNode graph D

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 1.0
AddEdge simpleGraph C D 1.0
AddEdge simpleGraph D B 1.0

AddEdge graph A B 3.456
AddEdge graph A C 1.534
AddEdge graph C D 1.563
AddEdge graph D B 0.545

FindPath simpleGraph A B

FindPath graph A B