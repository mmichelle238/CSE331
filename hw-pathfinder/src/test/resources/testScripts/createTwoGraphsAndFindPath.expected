# create multiple graphs and find paths between them
# simultaneously.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

created graph graph

added node A to graph
added node B to graph
added node C to graph
added node D to graph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from A to C in simpleGraph
added edge 1.000 from C to D in simpleGraph
added edge 1.000 from D to B in simpleGraph

added edge 3.456 from A to B in graph
added edge 1.534 from A to C in graph
added edge 1.563 from C to D in graph
added edge 0.545 from D to B in graph

path from A to B:
A to C with weight 1.000
C to D with weight 1.000
D to B with weight 1.000
total cost: 3.000

path from A to B:
A to B with weight 3.456
total cost: 3.456